napari/layers/surface/surface.py

```diff
@@ -578,7 +579,7 @@ def _has_texture(self) -> bool:
             and len(self.texcoords)
         )
 
-    def _get_state(self) -> dict:
+    def _get_state(self) -> _DeprecatingDict:
```

Comment:
maybe this PR should update this signature in all classes?

Reply:
I can do that.

After some other changes. I might actually be able to downgrade the annotate type to `dict[str, Any]` which I'd prefer. Then those really interested in the implementation details can go find out what type it is really.